autoupgrade.php

```diff
@@ -79,7 +79,14 @@ public function install()
             $tab = new Tab();
             $tab->class_name = 'AdminSelfUpgrade';
             $tab->module = 'autoupgrade';
-            $tab->id_parent = (int) Tab::getIdFromClassName('AdminTools');
+
+            // We have to check if 'DEFAULT' tab class name exist (introduced in V9) if not we use the old one 'AdminTools'
+            if (Tab::getIdFromClassName('DEFAULT')) {
+                $tab->id_parent = (int) Tab::getIdFromClassName('DEFAULT');
+            } else {
+                $tab->id_parent = (int) Tab::getIdFromClassName('AdminTools');
+            }
```

Comment:
If this version of the module is compliant with PS 1.7+ only, you may remove the `if` and only use the `DEFAULT` tab.

`DEFAULT` is a reference to deprecated tabs. It may be interesting to find a better place in the side bar to display the module in another PR.

Reply:
Thank you for your details. I will make the modification immediately. However if you have a better parent tab to put it in I would be happy to change it.

Reply:
Let's stick to the fix here. Moving the menu entry to another place should be a dedicated topic :)